BraveWallet/Extensions/BraveWalletExtensions.swift

```diff
@@ -196,8 +196,19 @@ extension BraveWallet.NetworkInfo {
 }
 
 extension BraveWallet.BlockchainToken {
-  /// The id to fetch price and price history. Use `coingeckoId` when it's not empty, otherwise use `symbol`
   var assetRatioId: String {
-    coingeckoId.isEmpty ? symbol : coingeckoId
+    if !coingeckoId.isEmpty {
+      return coingeckoId
+    }
+    
+    if chainId != BraveWallet.MainnetChainId {
```

Comment:
I think can remove below check and update this to include it? `if chainId != BraveWallet.MainnetChainId || contractAddress.isEmpty {`